packages/amplication-git-utils/src/git/github.service.ts

```diff
@@ -486,19 +491,27 @@ export class GithubService {
       // @ts-ignore
       tree: changesArray,
     });
+
+    console.log(`Created tree for for ${owner}/${repo}`);
```

Comment:
Please use a logger. This `console.log` usage floods the logs and bumps up bills on AWS CloudWatch and previously on Logz.io because we cannot shut them off in certain environments.

Also, logs like this should be structured.